docs/install/configuration.md

```diff
@@ -112,6 +112,7 @@ For more information about this feature, see [here](/docs/admin/telemetry.md)
 Option        | Description
 --------------|------------
 `telemetry.enabled` | Enables the anonymous usage telemetry of the platform. Default: `true`
+`telemetry.frontend.posthog.apihost` | The API host for PostHog, either 'https://app.posthog.com' or 'https://eu.posthog.com'. Default: `null`
```

Comment:
```suggestion
`telemetry.frontend.posthog.apihost` | The API host for PostHog, either 'https://app.posthog.com' or 'https://eu.posthog.com'. Default: `https://app.posthog.com`
```

We need to default to `https://app.posthog.com` otherwise things will break if this is deployed without the configuration made at the same time.